src/commands/init.ts

```diff
@@ -246,6 +249,10 @@ export async function initAction(feature: string, options: Options): Promise<voi
   if (!fsutils.fileExistsSync(config.path(".gitignore"))) {
     config.writeProjectFile(".gitignore", GITIGNORE_TEMPLATE);
   }
+  const duration = Math.floor((process.uptime() - start) * 1000);
+
+  await trackGA4("product_init", { products_initialized: setup.features?.join(",") }, duration);
```

Comment:
Emm, will this be easy to parse in GA?

We may have `firestore,dataconnect`, `dataconnect`, `dataconnect,storage` all mixed up.

It will be hard to find number of Firestore init for example.

Does GA support a list of labels?